HLTrigger/HLTfilters/plugins/L1TTkMuonFilter.cc

```diff
@@ -34,6 +60,8 @@ L1TTkMuonFilter::L1TTkMuonFilter(const edm::ParameterSet& iConfig)
   min_N_ = iConfig.getParameter<int>("MinN");
   min_Eta_ = iConfig.getParameter<double>("MinEta");
   max_Eta_ = iConfig.getParameter<double>("MaxEta");
+  applyQuality_ = iConfig.getParameter<bool>("applyQuality");
+  doDupRemoval_ = iConfig.getParameter<bool>("doDupRemoval");
```

Comment:
I'd prefer `applyQualityCuts` and `applyDuplicateRemoval` for consistency among the new parameter names

not a strong opinion, though

Reply:
I dont really mind. I'm happy to make that change.